Modified Sprint and GitHub master roles

RichardLitt opened the pull request:
Modified the sprint masters role, making it clear that they only run the sprint. The GitHub Masters role has been expanded to include making the sprint issue and all of the etherpads for the discussion leads, and pinging both leads beforehand and any active contributors afterwards. 

Also some basic edits.

RichardLitt commented:
Editing this.

jbenet commented on README.md:
```diff
@@ -45,21 +45,23 @@ The **Sprint Planning** meeting happens semi-synchronously. Some of it is synchr
 (This used to happen over a Google Hangout, but it wasted a lot of time to keep >10 people synchronized while people gave individual updates that not everyone was interested in. IRC has been working ok.)
 
 ##### The sprint master
-- Prepares the sprint's etherpad and posts it to GitHub and IRC ahead of time. To open an Etherpad, go to https://etherpad-mozilla.org.
```
no, i think the sprint master needs to prepare the discussion etherpads-- otherwise they will not be done.

jbenet replied:
well, someone has to. not each of the discussion leads

jbenet replied:
Ok i see https://github.com/ipfs/pm/pull/66/files#diff-04c6e90faac2675aa89e2176d2eec7d8R110 -- the distinction between these is unclear -- if these are meant to be separate roles (that's fine by me) -- "sprint master" should not be in the name of both...

RichardLitt replied:
Basically, one person should be in charge of the actual sync - making sure it runs, on IRC - and another person should be in charge of the GitHub sprint and etherpads. (Realistically, they're both me). I'll change the terminology to make it less confusing.

RichardLitt commented:
Ok. Overhauled it again. The real issue here is that we don't actually _have_ a Sprint master at the moment. We have an admin person doing stuff on GitHub, and a sync moderator, but not really someone who makes sure that everyone is up to date and that backlogs get pushed out. We do this as a team, and the discussion leads generally tend to be pseudo-autonymous in their particular areas. I've updated the document to show that, renamed sync master to sync moderator, clarified about the sprint master by creating a role for it, and also moved all of the roles down to their own section. I'm happier with this edition. What do yall think?

Dignifiedquire commented on README.md:
```diff
@@ -92,7 +87,7 @@ Time (PDT - **UTC/Z** - CET) | Topic
 You can add these meeting times to your Calendar app by adding our [community calendar](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84@group.calendar.google.com&ctz=America/New_York).
 
-The Hangouts are generated anew for each session. Sprint leaders should use [https://hangouts.google.com/start](https://hangouts.google.com/start) to make a new Hangout, and will drop the link in the IRC channel. 
+The Hangouts are generated anew for each session. Sprint leaders should use [https://hangouts.google.com/start](https://hangouts.google.com/start) to make a new Hangout, and will drop the link in the IRC channel.
```
This is not up to date anymore with the hangouts on air

Dignifiedquire commented on README.md:
```diff
@@ -118,9 +110,37 @@ To maximize feedback during a sprint, we've picked a set of "working hours" duri
-### Active Contributors
+### Roles
+
+#### The Sprint Master
```
Who is this currently, you @RichardLitt ? maybe make a note about that somewhere?

RichardLitt replied:
Added a note: yeah, it's me.

RichardLitt commented:
Overhauled again: I just merged GitHub, Sync, and Discussion moderator into one role, the admin, which really is me. This reads a bit cleaner now. 

CR? cc @Dignifiedquire @jbenet

Dignifiedquire commented:
LGTM

RichardLitt commented:
Thanks. I am going to merge this, as this only reflects the processes we already have in place. If anyone wants to discuss this post merge, open a PR or ping me directly.
